### ISSUE TITLE
possible issues with package name?

### ISSUE THREAD

**Issue by cboettig** [ropenscilabs/datapkg#1]
Just a quick note that `ropensci` has had a repo called `datapackage` for some time, which was recently renamed to `datapack` at the request of CRAN. see: https://github.com/ropensci/datapack/issues/46

**Comment by karthik** [comment]
Good call. We should rename it and also move it to labs for the time being.

**Comment by jeroen** [comment]
Ah thanks for catching this.

**Comment by jeroen** [comment]
FYI I can't transfer this to labs:

> You don’t have admin rights to ropenscilabs

**Comment by richfitz** [comment]
I moved it over

**Comment by jeroen** [comment]
OK now we just need a better name =)

**Comment by rufuspollock** [comment]
@jeroenooms @cboettig what are the CRAN issues with `datapackage`? Is it already taken as a package name?

**Comment by cboettig** [comment]
@rgrp see https://github.com/ropensci/datapack/issues/46; basically CRAN maintainers didn't like the name (when the different but closely related package at that github link was submitted), and they're the gatekeepers.  It's not unimaginable that you could submit this package with the same name and get it through though, gatekeeping has not been proven to be a deterministic process.

on the other hand, users might already wonder how the existing ropensci package "datapack" differs from one called "datapackage", so I'd suggest a name that more obviously conveys the standard to which said datapackage conforms, i.e. maybe something that references okfn or something?

**Comment by rufuspollock** [comment]
@cboettig it is not Open Knowledge specific so I don't think we would want that necessarily in the name IMO (and we have not done that for other languages). Maybe we try `datapkg` if `datapackage` is impossible.
